src/Integration/Suppression/SuppressedIssuesFileStorage.cs

```diff
@@ -0,0 +1,113 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Abstractions;
+using System.Linq;
+using Newtonsoft.Json;
+using SonarLint.VisualStudio.Core.Suppressions;
+using SonarQube.Client.Models;
+
+namespace SonarLint.VisualStudio.Integration.Suppression
+{
+    internal class SuppressedIssuesFileStorage : ISuppressedIssuesFileStorage
+    {
+        private readonly string fileDirectory;
+        private readonly ILogger logger;
+        private readonly IFileSystem fileSystem;
+
+        public SuppressedIssuesFileStorage(ILogger logger) : this(logger, new FileSystem())
+        {
+            
+        }
+
+        internal SuppressedIssuesFileStorage(ILogger logger, IFileSystem fileSystem)
+        {
+            this.fileSystem = fileSystem;
+            this.logger = logger;
+
+            fileDirectory = Path.Combine(Path.GetTempPath(), "SLVS", "Roslyn");
+        }
+
+        public IEnumerable<SonarQubeIssue> Get(string sonarProjectKey)
```

Comment:
The information in the file needs to be passed to the diagnostic suppressor, so doesn't this class need to be in the new assembly?

Reply:
All the other interface implementations for same namespace was here so I've added this also here to be consistent.

Reply:
How is the diagnostic suppressor going to use this type?